docs/src/index.md

```diff
@@ -5,16 +5,100 @@ matrices that have uncertain parameters.
 
 ## Features
 
-- An `IntervalMatrix` type.
-- Arithmetics between scalars, closed intervals and interval matrices.
-- Overapproximation and underapproximation routines for the exponential of an
-  interval matrix at different orders.
+Here is a quick summary of the available functionality.
+See the section `Library Outline` below for details.
 
-## Library Outline
+- An `IntervalMatrix` type that wraps a two-dimensional array whose components
+  are intervals.
+- Arithmetic between scalars, intervals and interval matrices.
+- Quadratic expansions using single-use expressions (SUE).
+- Interval matrix exponential: underapproximation and overapproximation routines.
+- Utility functions such as: operator norm, random sampling, splitting and
+  containment check.
+
+## Quickstart
+
+An *interval matrix* is a matrix whose coefficients are intervals. For instance,
+
+```julia
+julia> A = IntervalMatrix([0..1 1..2; 2..3 -4.. -2])
```

Comment:
Should these not be `jldoctests` so we are informed if something changes?

Reply:
yes, that's a good idea